superset/db_engine_specs/base.py

```diff
@@ -1057,30 +1057,33 @@ def extract_errors(
         ]
 
     @classmethod
-    def adjust_database_uri(  # pylint: disable=unused-argument
+    def adjust_engine_params(  # pylint: disable=unused-argument
```

Comment:
@betodealmeida what do you think will be the impact of this change on any custom db engine specs that have been added to Superset instances?

Reply:
oh, nm, I see how you covered this below.